lighthouse-core/report/html/renderer/util.js

```diff
@@ -26,6 +26,40 @@ class Util {
     return PASS_THRESHOLD;
   }
 
+  static get MS_DISPLAY_VALUE() {
+    return `%10d${NBSP}ms`;
+  }
+
+  /**
+   * @param {string|Array<string|number>=} displayValue
+   * @return {string}
+   */
+  static formatDisplayValue(displayValue) {
+    if (typeof displayValue === 'string' || typeof displayValue === 'undefined') {
+      return displayValue || '';
+    }
+
+    const template = /** @type {string} */ displayValue.shift();
+    if (typeof template !== 'string') {
+      // First value should always be the format string, but we don't want to fail to build
+      // a report, return a placeholder.
+      return 'UNKNOWN';
+    }
+
+    let output = template;
+    while (displayValue.length) {
+      const replacement = /** @type {number|string} */ (displayValue.shift());
```

Comment:
```js
    assert.equal(format(['%s']), '%s', 'not enough replacements');
    assert.equal(format(['%s', 'a', 'a']), 'a', 'extra replacements');
```

i think we may want to console.warn() in these cases. Allow them, but raise a warning flag.